mooseutils/validate.py

```diff
@@ -0,0 +1,46 @@
+import os
+import sys
+import errno
+import logging
+
+def validate_extension(*args, extension, raise_on_error=False, log_on_error=True):
```

Comment:
Can you add unit tests for these in mooseutils/tests, call it "test_validate.py"

Reply:
I already did, it is in mooseutils/tests :)